client/my-sites/marketing/connections/mailchimp-settings.jsx

```diff
@@ -62,7 +64,7 @@ const MailchimpSettings = ( {
 			<div className="sharing-connections__mailchimp-gutenberg_explanation">
 				<p>
 					{ translate(
-						'Start building your mailing list by adding the Mailchimp block to your posts and pages. '
+						'Start building your audience by adding the Mailchimp block to your posts and pages. '
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.